src/lib/user_command_input/user_command_input.ml

```diff
@@ -186,14 +188,58 @@ let to_user_command ?(nonce_map = Account_id.Map.empty) ~get_current_nonce
       ~inferred_nonce
     |> Deferred.return
   in
+  let receiver_pk = Signed_command_payload.receiver_pk user_command_payload in
+  let token = Signed_command_payload.token user_command_payload in
+  (* TODO(omer): is this the "right" token? *)
+  let receiver : Account_id.t = Account_id.create receiver_pk token in
+  let receiver_account = get_account receiver in
+  (* If the receiver account doesn't exist yet (as far as we can tell) *and* the
+   * user command isn't sufficient to cover the account creation fee, log a
+   * warning. *)
+  let () =
+    match receiver_account with
+    | `Bootstrapping ->
+        () (* TODO(omerzach): What should we do here? *)
```

Comment:
When bootstrapping should we just skip this check? Or should we never be in this case?

Reply:
yeah just skip in this case, there's nothing else we can do